packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -15,6 +15,8 @@ export enum WalletsMethod {
   Delete = 'delete',
   GetActive = 'getActive',
   Activate = 'activate',
+  RenameWallet = 'renameWallet',
```

Comment:
`rename` and `update addresses` belong to `update`

Reply:
fixed